### PR TITLE
Remove SeekingQueryPlanner code.

### PR DESCRIPTION
**Pull request by jack-gitdev**

### CODE REVIEW

File: warehouse/query-core/src/main/java/datawave/query/config/LookupUUIDTune.java [hunk 1]
```diff
@@ -75,12 +74,9 @@ public void configure(BaseQueryLogic<Entry<Key,Value>> logic) {
...
                 
-                // setup SeekingQueryPlanner in case the queryIterator requires it
-                SeekingQueryPlanner planner = new SeekingQueryPlanner();
-                planner.setMaxFieldHitsBeforeSeek(maxFieldHitsBeforeSeek);
-                planner.setMaxKeysBeforeSeek(maxKeysBeforeSeek);
-                
-                rsq.setQueryPlanner(planner);
+                // pass through seek options
+                rsq.setFiFieldSeek(maxFieldHitsBeforeSeek);
+                rsq.setFiNextSeek(maxKeysBeforeSeek);
```
Comment (apmoriarty) [inline warehouse/query-core/src/main/java/datawave/query/config/LookupUUIDTune.java]: `maxFieldHitsBeforeSeek` and `maxKeysBeforeSeek` should be removed along with any getters/setters. 

Then you can set the fi and event seek values from lines 34-37

**Review by apmoriarty — CHANGES_REQUESTED** [review]

**Review by apmoriarty — DISMISSED** [review]

**Review by avgAGB — DISMISSED** [review]

**Review by jwomeara — COMMENTED** [review]
I'm guessing the answer is "because we don't use it", but why are we removing this?

**Comment by apmoriarty** [comment]
> I'm guessing the answer is "because we don't use it", but why are we removing this?

The functionality of the SeekingQueryPlanner was pushed into the DefaultQueryPlanner in these two merge requests. This merge request deals with cleaning up the now-dead code.
- https://github.com/NationalSecurityAgency/datawave/pull/1869
- https://github.com/NationalSecurityAgency/datawave/pull/1883

**Comment by jwomeara** [comment]
> > I'm guessing the answer is "because we don't use it", but why are we removing this?
> 
> The functionality of the SeekingQueryPlanner was pushed into the DefaultQueryPlanner in these two merge requests. This merge request deals with cleaning up the now-dead code.
> 
> * [More ways to configure seeking in the QueryIterator #1869](https://github.com/NationalSecurityAgency/datawave/pull/1869)
> * [Integrate seeking configuration changes #1883](https://github.com/NationalSecurityAgency/datawave/pull/1883)

Perfect!  That's the piece of information I was missing.

**Review by jwomeara — DISMISSED** [review]

**Review by apmoriarty — DISMISSED** [review]

**Review by avgAGB — DISMISSED** [review]

**Review by hlgp — APPROVED** [review]

**Review by alerman — APPROVED** [review]

**Review by apmoriarty — APPROVED** [review]

**Pull request merged**